packages/report-server/src/reportBuilder/transform/functions/index.ts

```diff
@@ -47,21 +48,15 @@ export const transformBuilders: Record<string, TransformBuilder> = {
   orderColumns: buildOrderColumns,
 };
 
-export const transformSchemas: Record<
-  string,
-  {
-    type: string;
-    fields: Record<string, unknown>;
-  }
-> = {
-  fetchData: fetchDataParamsValidator.describe(),
-  insertColumns: insertColumnsParamsValidator.describe(),
-  excludeColumns: excludeColumnsParamsValidator.describe(),
-  updateColumns: updateColumnsParamsValidator.describe(),
-  mergeRows: mergeRowsParamsValidator.describe(),
-  sortRows: sortRowsParamsValidator.describe(),
-  excludeRows: excludeRowsParamsValidator.describe(),
-  insertRows: insertRowsParamsValidator.describe(),
-  gatherColumns: gatherColumnsParamsValidator.describe(),
-  orderColumns: orderColumnsSchema,
+export const transformSchemas: Record<string, yup.AnyObjectSchema> = {
```

Comment:
Had to switch to passing in the schema itself, since it's required to get the default value (it's not part of the schema description)